build_tools/bazel/iree_e2e_generated_runner_test.bzl

```diff
@@ -28,8 +29,9 @@ def iree_e2e_matmul_test(
 
     Args:
         name: Name of the target
-        matmul_src: mlir source file with matmuls to be compiled.
-        matmuls_vmfb: specifies the path to use for the generated IREE module.
+        test_type: Name of the test (e.g., matmuls, conv2ds).
```

Comment:
nit: plural feels weird here? could just have `matmul` and `conv2d` as the names

Reply:
Test_type has been changed to 'matmul' and 'conv2d' instead of the usual.